packages/element-web-module-api/src/api/extras.ts

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2025 New Vector Ltd.
+
+SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-Element-Commercial
+Please see LICENSE files in the repository root for full details.
+*/
+
+import { JSX } from "react";
+
+/**
+ * Properties of an item added to the Space panel
+ * @alpha
+ */
+export interface SpacePanelItemProps {
+    /**
+     * A CSS class name for the item
+     */
+    className?: string;
+
+    /**
+     * An icon to show in the item
+     */
+    icon?: JSX.Element;
+
+    /**
+     * The label to show in the item
+     */
+    label: string;
+
+    /**
+     * A tooltip to show when hovering over the item
+     */
+    contextMenuTooltip?: string;
```

Comment:
this naming is confusing, what does the tooltip have to do with context menus if it is shown when hovering over the item?